### DISCUSSION TITLE
Option to copy the header files with install_headers if they are symbolic links?

### DISCUSSION THREAD

**Discussion by amontoison** [mesonbuild/meson#11471] [Q&A]
I have the following warning when I install the header files because they are symbolic links.
The good news is that it's the behaviour that I want!
What should I modify to always copy the files, such that it will not break in the future?
```
Warning: trying to copy a symlink that points to a file. This will copy the file,
but this will be changed in a future version of Meson to copy the symlink as is. Please update your
build definitions so that it will not break when the change happens.
```

**Comment by eli-schwartz** [comment]
@ArsenArsen is trying to deal with this in #10594, because it seems plausible to allow either form.

**Reply by ArsenArsen** [reply]
yeah, we have a very similar use-case.  I can rebase that PR tomorrow and extend it to other install_ functions, if you think that's the way to go, @eli-schwartz.
